includes/Products/FBCategories.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Products;
+
+defined( 'ABSPATH' ) or exit;
+
+use SkyVerge\WooCommerce\PluginFramework\v5_5_4 as Framework;
+
+/**
+ * The main product feed handler.
+ *
+ * This will eventually replace \WC_Facebook_Product_Feed as we refactor and move its functionality here.
+ *
+ * @since 1.11.0
+ */
+class FBCategories {
```

Comment:
This is the class that does most of the work loading and maintaining the attributes. It also sets up the hooks for the ajax calls.